recipes-bsp/rpi-eeprom/rpi-eeprom_git.bb

```diff
@@ -0,0 +1,71 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
```

Comment:
This is not needed.

Reply:
@agherzan I hope all your suggestions are fixed now.

Reply:
Thanks. Added some more comments. Also, a rebase is needed (and a test).